lib/react_client/react_interop.dart

```diff
@@ -57,39 +57,43 @@ abstract class React {
 ///     }
 ///
 /// Learn more: <https://reactjs.org/docs/refs-and-the-dom.html#creating-refs>.
-Ref<CurrentType> createRef<CurrentType>() {
-  return new Ref<CurrentType>();
+Ref<T> createRef<T>() {
+  return new Ref<T>();
```

Comment:
#supernit

The `new` keyword is no longer necessary

Reply:
Yeah at some point we should just go through and remove all of these